frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/ParametersSettings.tsx

```diff
@@ -95,9 +120,16 @@ export const ParametersSettings = ({
                   className="m0"
                   parameter={parameter}
                   parameters={valuePopulatedLockedParameters}
-                  setValue={(value: string) => {
-                    onChangeParameterValue(parameter.id, value);
+                  setValue={(value: string) =>
+                    onChangeParameterValue(parameter.id, value)
+                  }
+                  setParameterValueToDefault={() => {
+                    onChangeParameterValue(
+                      parameter.id,
+                      parameter.default as any,
```

Comment:
why any, can it be converted to string?

Reply:
It can't, parameters are often arrays of values.

Reply:
okay, but can you come up with a better type than `any`?

Reply:
I definitely will — not that I am discarding your suggestion.
But if you don't mind, as a part of a separated effort to tidy up the parameters everywhere 🙏